orttraining/orttraining/python/training/torchdynamo/__init__.py

```diff
@@ -2,3 +2,12 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 # --------------------------------------------------------------------------
+
+from typing import Set
+
+# set of custom ops supported in DORT
```

Comment:
```suggestion
# set of custom ops supported in DORT.
# It can contain, for example, `aten::custom_add`.
```
Always consider some concrete examples for elements in a collection object.